capture/noworkflow/now/models/definition.py

```diff
@@ -0,0 +1,75 @@
+# Copyright (c) 2016 Universidade Federal Fluminense (UFF)
+# Copyright (c) 2016 Polytechnic Institute of New York University.
+# This file is part of noWorkflow.
+# Please, consult the license terms in the LICENSE file.
+"""Definition Object"""
+from __future__ import (absolute_import, print_function,
+                        division, unicode_literals)
+
+from collections import OrderedDict
+
+from future.utils import viewkeys
+
+from ..persistence.models.base import Model, proxy_gen
+from ..persistence.models.trial import Trial
+from .graphs.definition_graph import DefinitionGraph, DefinitionAst
+
+
+class Definition(Model):
+    """This model represents a definition of trials
+    Initialize it by passing a trials id:
+        definition = Definition(1)
+
+    There are one visualization modes for the graph:
+        definition_tree: definition tree without any filters
+            definition.graph.mode = 0
+
+
+    You can change the graph width and height by the variables:
+        definition.graph.width = 600
+        definition.graph.height = 400
+    """
+
+    __modelname__ = "Definition"
+
+    DEFAULT = {
+        "graph.width": 500,
+        "graph.height": 500,
+        "graph.mode": 0
+    }
+
+    REPLACE = {
+        "graph_width": "graph.width",
+        "graph_height": "graph.height",
+        "graph_mode": "graph.mode"
+    }
+
+    def __init__(self, trial_ref, **kwargs):
+        super(Definition, self).__init__(trial_ref, **kwargs)
+        self.trial = Trial(trial_ref)
```

Comment:
In some situations we may have the Trial object instead of the trial_ref. It would be good to be able to initialize the Definition both by passing a trial_ref or a Trial object and have this object without reinstantiating Trial

Reply:
Is it necessary that trial_ref (or kwargs) provides a Trial object? Because when calling the trial's ID, I use `self.trial.trial.id`.

Reply:
As for now, I moved the definition into trial_graph